src/renderer/components/publication/TagManager.tsx

```diff
@@ -68,15 +70,24 @@ export class TagManager extends React.Component<Props, TagManagerState> {
                     )}
                 </ul>}
                 {this.props.canModifyTag &&
-                    <form onSubmit={this.addTag} id={styles.flux_search}>
+                    <form id={styles.flux_search}>
                         <input
                             type="text"
                             className={styles.tag_inputs}
                             title={ __("catalog.addTags")}
                             placeholder={ __("catalog.addTags")}
                             onChange={this.handleChangeName}
                             value={this.state.nameNewTag}
+                            onKeyDown={(e) => {
+                                if (e.keyCode === 13) {
+                                    e.preventDefault();
+                                    return (false);
+                                }
+                            }}
```

Comment:
I think we should preserve the original behaviour (i.e. `enter` key submits the HTML form).